CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
 
 </details>
 
+## 1.47.1 (May 7, 2020)
+
```

Comment:
Oh, no need to add the date here since we haven't released it yet. This should go under the "unreleased" part.

Reply:
```## [Unreleased]

## 1.47.1

### Patch

- VideoControls: overwriting overflow default behavior on timestamp text (#845)```

Reply:
Like this?